Source/GSXib5KeyedUnarchiver.m

```diff
@@ -170,6 +169,24 @@ @implementation GSXib5KeyedUnarchiver
 static NSArray      *XmlConnectionRecordTags  = nil;
 static NSArray      *XmlBoolDefaultYes  = nil;
 
+@interface NSString (XibExtension)
+
+- (BOOL) hasPrefixedClassName;
```

Comment:
We should not in our library clutter up the namespace of classes from Foundation if this could be avoided easily.

Reply:
Agreed, I'll do this another way.  I believe it can be undone at this point.

Reply:
I may take this out if it's not needed.

Reply:
Removed.